vector/v.category/testsuite/test_v_category.py

```diff
@@ -0,0 +1,23 @@
+import json
+
+from grass.gunittest.case import TestCase
+from grass.gunittest.gmodules import call_module
+
+
+class TestVCategory(TestCase):
+
+    def test_d_flag(self):
+        expected = [
+            {"type": "point", "field": 1, "count": 10938, "min": 1, "max": 10938},
+            {"type": "all", "field": 1, "count": 10938, "min": 1, "max": 10938},
+        ]
+        output = call_module(
+            "v.category", input="bridges", option="report", format="json"
+        )
+        self.assertListEqual(expected, json.loads(output))
+
```

Comment:
The option `print` does not respect the format. Add a test case for `option="print", format="json"`. 

It can return an array of category numbers.

```json
{
   "categories": [1,2,3,4,5,..]
}
```

Reply:
The same should be done when the option is set to `layers`.